docs/src/components/Navbar/Links.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Hide, TextLink, Stack, Button, Popover } from "@kiwicom/orbit-components";
+import { MenuHamburger } from "@kiwicom/orbit-components/lib/icons";
+
+import PuzzleIcon from "../icons/Puzzle";
+import ColorBrushIcon from "../../images/streamline-light/color-brush.svg";
+import ReadArt from "../../images/streamline-light/read-art.svg";
+import IrisScanIcon from "../../images/streamline-light/iris-scan.svg";
+import ScriptIcon from "../../images/streamline-light/script.svg";
+
+type Link = "Components" | "Guides" | "Tokens" | "Accessibility" | "Foundation";
+
+interface Props {
+  links?: Link[];
+}
+
+const LinkIcon = ({
```

Comment:
Honestly I'm not sure if this is the best approach, in terms of readability/maintainability. Because it conditionally renders either an icon or the dot separating the links. Shouldn't it be just for the icon and then have the condition of either rendering the Icon (this component) or the dot on the `LinkList` component? (Suggestion below)